test/test-utils/spy.js

```diff
@@ -0,0 +1,48 @@
+// Inspired by https://github.com/popomore/spy
+export default function(obj, func) {
```

Comment:
Very handy. Any interest in open-sourcing this after we've given it some time to settle? (Would be nice to put it out there without having to bother with maintenance, perhaps we draw a hard line that it is a "micro-spy" ([name is unclaimed](https://www.npmjs.com/package/microspy)) and we're not looking for feature requests...